src/features/page_view_event/aggregate/index.js

```diff
@@ -97,23 +97,19 @@ export class Aggregate extends AggregateBase {
       chunksForQueryString.push(param('perf', stringify(navTimingApiData)))
     }
 
-    if (globalScope.performance && globalScope.performance.getEntriesByType) {
+    try { // PVTiming sends these too, albeit using web-vitals and slightly different; it's unknown why they're duplicated, but PVT should be the truth
```

Comment:
This adds FP and FCP to the `PageView` event, while PVT adds it to the `PageViewTiming` events.  If web-vitals is intended to be the truth, this needs to get updated to use web-vitals here if it can (and fallback like PVT does)

Reply:
If we make it so either isn't aware which one runs first (and so creates the single value in `paintMetrics`), it gets ugly-looking real quick with refactoring onFirstPaint & onFCP out into its own module and passing more layers of callbacks. In majority cases, they'll end up with the same value as-is anyhow. I'm trying not to complicate it.

Reply:
Can't you just call `onFCP(...)` in the PVE Agg as well?  That's what I was picturing for this

Reply:
If it's a hairy process lets just add a cleanup ticket for later. NBD

Reply:
I suppose we _could_ but also their API isn't exactly synchronous, so we'd have to add an async-ness into sendRum. Still not a big deal but pointing out it's ugly with duplicate-looking but also not easily refactor-able code.

Reply:
> Lets just add a cleanup/consolidation ticket for later. NBD -- 👍

Reply:
Yeah, will add a ticket for this, because my attempt turned it into this monstrosity, and it didn't even pass `tests/functional/rum/paint-timing.test.js`... 
<img width="1696" alt="image" src="https://user-images.githubusercontent.com/28714891/227397227-91549705-9fe6-44a5-b33f-2b14136838f8.png">

Reply:
https://issues.newrelic.com/browse/NR-101330